spec/compiler/interpreter/autocast_spec.cr

```diff
@@ -102,5 +102,38 @@ describe Crystal::Repl::Interpreter do
           foo.color.value
         CODE
     end
+
+    it "autocasts integer var to integer (#12560)" do
+      interpret(<<-CODE).should eq(1)
+        def foo(x : Int64)
+          x
+        end
+
+        x = 1
```

Comment:
I'd suggest to make the literal type explicit because it's important that the value is an `Int32`.
```suggestion
        x = 1_i32
```

Reply:
Sorry, but I don't understand. The value is an Int32. `1` is an `Int32` literal. But I don't want to lose time arguing, so I'll just accept it 😊

Reply:
Yes it is, but that's implicit knowledge. Adding `_32` makes it explicit, which I think helps understanding this spec. The type of the variable is a critical component.
Plus the spec will continue working in case we ever change the default number type to `Int64` (even though that's unlikely).